modulus/datapipes/climate/climate_hdf5.py

```diff
@@ -0,0 +1,510 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Do we want to keep both `climate_hdf5.py` and the existing `era5_hdf5.py`?

Reply:
This is a great question I will address it bellow with some explination.